.sauce/config.yml

```diff
@@ -11,13 +11,27 @@ sauce:
     build: Release $CI_COMMIT_SHORT_SHA
 
 xcuitest:
-  app: ./apps/SauceLabs-Demo-App.ipa
-  testApp: ./apps/SauceLabs-Demo-App-Runner.XCUITest.ipa
+  app: $APP
+  testApp: $TEST_APP
 
 suites:
   - name: "saucy xcuitest"
     testOptions:
```

Comment:
I'd recommend removing this line if it's empty.

Reply:
removed.